prow/cluster/gcsweb/gcsweb_ing.yaml

```diff
@@ -6,15 +6,10 @@ metadata:
   name: gcsweb-ing
   namespace: gcs
   annotations:
-    kubernetes.io/ingress.global-static-ip-name: k8s-fw-gcs-gcsweb-ing--c44bd72f22c73d37
+    kubernetes.io/ingress.global-static-ip-name: k8s-fw-gcs-gcsweb-ing--c44bd72f22c73d37, gcsweb
     networking.gke.io/managed-certificates: gcs-istio-io
     kubernetes.io/ingress.class: "gce"
 spec:
-  rules:
-  - host: gcsweb.istio.io
-    http:
-      paths:
-      - path: /*
-        backend:
-          serviceName: gcsweb
-          servicePort: 80
+  backend:
```

Comment:
Why not do the same for the velodrome ingress?

Reply:
I am. See: https://github.com/istio/test-infra/pull/2521/